.github/workflows/deploy_to_github_io.yml

```diff
@@ -4,11 +4,13 @@ on:
   push:
     branches:
       - main
+  workflow_dispatch:
```

Comment:
is this yaml key missing it's content?  not sure if this is valid yaml, or if it 'just works', https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/ seems to suggest you might not have any inputs or tags

Reply:
> You must append a colon (:) to all events, including events without configuration.

https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#example-using-multiple-events-with-activity-types-or-configuration